spec/features/observer_spec.rb

```diff
@@ -39,7 +39,7 @@
   it "allows a user to add a reason when adding an observer" do
     reason = "is the archbishop of banterbury"
     proposal = create(:proposal)
-    observer = create(:user)
+    observer = create(:user, client_slug: nil)
```

Comment:
oh I see, that's interesting. I think that updating the description of the below test from "identifies eligible observers"  to "identifies eligible observers based on the client slug" would be :+1: 

One reason it feels a little unexpected that there are so many records here with a nil client_slug is that we don't have actual users creating proposals / adding observers who have a `client_slug` of nil, right?